controllers/velero_test.go

```diff
@@ -712,7 +718,9 @@ func TestDPAReconciler_buildVeleroDeployment(t *testing.T) {
 									Command: []string{"/velero"},
 									Args: []string{
 										"server",
-										"--fs-backup-timeout=1h",
+										"--log-level=info",
```

Comment:
I did not like to change `--fs-backup-timeout=1h` to `--fs-backup-timeout=1h0m0s`, but I think is fine. But Isn't wrong to have to have to log level flags in some tests?

Reply:
I wonder whether you should remove the log-level without info below? default level is info in any case. I think this is probably fine as-is if tests pass, but it may be worth finding a way to remove the duplicate log-level param.

Reply:
Found out, here https://github.com/openshift/oadp-operator/blob/master/controllers/velero.go#L308 (through here https://github.com/openshift/oadp-operator/blob/master/controllers/velero.go#L437) was triggering the addition of "--log-level=info" (and also changing 1h to 1h0m0s), but here https://github.com/openshift/oadp-operator/blob/master/controllers/velero.go#L317 does the exact same check of log, so it is safe to remove it from `StringArr` function, right?

Reply:
going to ignore refactor needs for now